types/src/crypto/asymmetric_key.rs

```diff
@@ -287,7 +281,7 @@ impl Display for PublicKey {
             formatter,
             "PubKey::{}({:10})",
             self.variant_name(),
-            HexFmt(Into::<Vec<u8>>::into(self))
+            HexFmt(&Into::<Vec<u8>>::into(self))
```

Comment:
```suggestion
            HexFmt(Into::<Vec<u8>>::into(self))
```

Reply:
Fixed in [e93dc37](https://github.com/nilz3ro/casper-node/pull/5/commits/e93dc3787a9983c352ebfdba0fc6bccb8f4d7c41).